src/geometries/CapsuleGeometry.js

```diff
@@ -0,0 +1,49 @@
+import { Path } from '../extras/core/Path';
+import { LatheGeometry } from './LatheGeometry';
+
+class CapsuleGeometry extends LatheGeometry {
+
+	/**
+	 * Creates a CapsuleGeometry.
+	 *
+	 * @param {number} radiusTop — Radius of the emisphere at the top. Optional; defaults to 1.
+	 * @param {number} radiusBottom — Radius of the emisphere at the bottom. Optional; defaults to 1.
+	 * @param {number} height — Height of the middle section. Optional; defaults to 1.
+	 * @param {number} capSegments — Number of curve segments used to build the caps. Optional; defaults to 4.
+	 * @param {number} radialSegments — Number of segmented faces around the circumference of the capsule. Optional; defaults to 8.
+	 */
```

Comment:
Should I remove these? The rest of the codebase doesn't use them so it feels irrelevant

Reply:
We can remove these yeah.